### ISSUE TITLE
wrapper.api.NewBookTickerService undefined and wrapper.api.NewPriceChangeStatsService undefined

### ISSUE THREAD

**Issue by Megavolv** [saniales/golang-crypto-trading-bot#91]
Hello!

I can't compile app.

golang/src/github.com/saniales/golang-crypto-trading-bot/exchanges/binance.go:191:35: wrapper.api.NewBookTickerService undefined (type *binance.Client has no field or method NewBookTickerService)
golang/src/github.com/saniales/golang-crypto-trading-bot/exchanges/binance.go:209:37: wrapper.api.NewPriceChangeStatsService undefined (type *binance.Client has no field or method NewPriceChangeStatsService)

Can you fix it?

**Comment by saniales** [comment]
It seems that the binance library has been updated and the bot does not work anymore. I will check it

**Comment by skycube** [comment]
This is not related directly to `binance` itself, the imported library which is not directly`binance` `https://github.com/adshao/go-binance/releases` had a mayor update to implement a `v3` of binance api rest endpoints.
If you checkout `1.1.2` in the above library, you be able to compile again.
aka 
```
cd $GPATH/src/github.com/adshao/go-binance;
git checkout 1.1.2;
```

**Comment by Megavolv** [comment]
Thank you very-very much!!!

:))


>Понедельник, 21 октября 2019, 4:03 +03:00 от Per Lasse Baasch <notifications@github.com>:
>
>This is not related directly to binance itself, the imported library which is not directlybinance  https://github.com/adshao/go-binance/releases had a mayor update to implement a v3 of binance api rest endpoints.
>If you checkout 1.1.2 in the above library, you be able to compile again.
>aka
>cd $GPATH/src/github.com/adshao/go-binance;
>git checkout 1.1.2;
>
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe .

-- 
С уважением,
Петр Филонов

**Comment by saniales** [comment]
This will be solved officially with go module update in this repo too, thanks @skycube anyway

**Comment by saniales** [comment]
Closed by #94 and #95
